Fredrik_Grondahl.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fredrik</title>
+</head>
+<body>
+    <h1> Fredrik Grondahl</h1>
+    <p>fredrikgr@hotmail.com</p>
+    
```

Comment:
lägg till länk tillbaka till index och en lista med intressen

Reply:
Will do ... thank you for reminding!